### PR TITLE
Ci-add-premerge-tests

### PR DESCRIPTION
**Pull request by DaveSys911**
What does this pr do?

- Removes a k8s API test cases that hits a k8s auth mode bug
- Skip of flaky file system test
- Fix publish results action
- Adding additional test suites to pre-merge.

FS will be optional for now, due to a few flaky tests that needs addressing.
The rest will be set as required.

### CODE REVIEW

**Comment by DaveSys911** [comment]
Allure link: http://csp-allure-reports.s3.amazonaws.com/allure_reports/cloudbeat/allure_reports/cloudbeat/prs/pr-/index.html

**Comment by DaveSys911** [comment]
Allure link: http://csp-allure-reports.s3.amazonaws.com/allure_reports/cloudbeat/allure_reports/cloudbeat/prs/pr-274/index.html

File: tests/product/tests/data/process/process_test_cases.py [hunk 1]
```diff
@@ -468,18 +468,7 @@
         '/etc/kubernetes/manifests/kube-apiserver.yaml',
         'passed'
     )]
-
 cis_1_2_8 = [(
```
Comment (oren-zohar) [inline tests/product/tests/data/process/process_test_cases.py:471]: why did you remove the test? should we just skip it?

Comment (DaveSys911) [reply tests/product/tests/data/process/process_test_cases.py:471]: The RBAC auth mode for node communication is something that is deprecated from 1.18.
Kind won't support it out of the box. So setting it causes 500 errors from the API server.

File: .github/workflows/publish-test-results.yml [hunk 2]
```diff
@@ -51,20 +60,23 @@ jobs:
...
           ignoreMissingResults: true
 
 # Todo Workflow_run jobs only work from default branch.Verify post merge pr processing is functional & secure gh ticket https://github.com/elastic/security-team/issues/3936
+      - name: Output to Job Summary
+        if: ${{ github.event.workflow_run.event == 'pull_request' }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "### Run Summary :rocket:" >> $GITHUB_STEP_SUMMARY
```
Comment (oren-zohar) [inline .github/workflows/publish-test-results.yml:73]: 🚀

File: .gitignore [hunk 3]
```diff
@@ -13,18 +13,24 @@ cloudbeat-test
 *local.yaml
 *local.yml
 fields.yml
-tests/allure/results/*
-tests/allure/results/test.dummy
-tests/allure/reports/*
```
Comment (oren-zohar) [inline .gitignore:18]: 💯

**Review by oren-zohar — APPROVED** [review]
LGTM

**Comment by amitkanfer** [comment]
🚀
